index.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+require('vendor/autoload.php');
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+use Nexmo\Client;
+use Nexmo\Client\Credentials\Keypair;
+use Aws\S3\S3Client;
+use League\Flysystem\AwsS3v3\AwsS3Adapter;
+use League\Flysystem\Filesystem;
+use Aws\TranscribeService\TranscribeServiceClient;
+
+Dotenv\Dotenv::create(__DIR__)->load();
+
+$app = AppFactory::create();
+
+/**
+ * Answer the call and prompt using NCCO flow
+ */
+$app->any('/webhooks/answer', function (Request $request, Response $response) {
+    $uri = $request->getUri();
+
+    if ($request->getMethod() != 'GET') {
+        return $response->withStatus(403);
+    }
+
+    // NOTE: The usage of $uri below will not work when using ngrok, as getHost() will return localhost
+    $ncco = [
+        [
+            'action' => 'talk',
+            'text' => 'Please leave a message after the tone, then press #.'
+        ],
+        [
+            'action' => 'record',
+            'eventUrl' => [
+                $uri->getScheme().'://'.$uri->getHost().':'.$uri->getPort().'/webhooks/fetch'
+            ],
+            'endOnSilence' => '3',
+            'endOnKey' => '#',
+            'beepOnStart' => true
+        ],
+        [
+            'action' => 'talk',
+            'text' => 'Thank you for your message. Goodbye.'
+        ],
+        [
+            'action' => 'notify',
+            'payload' => ['foo'=>'bar'],
```

Comment:
What is this payload being used for?

Reply:
Not used for anything, but thought it was required.

Reply:
This is actually a good question, because the docs do say it's required. Maybe we should change this to a payload that actually does something, like sending back the MSISDN or some other "customer" data to show how we can track the events better? That will look better than a dummy value that's just arbitrarily passed back to satisfy the API.

Reply:
Set it as a followup flag for the example.